datamodel_code_generator/__main__.py

```diff
@@ -326,141 +355,151 @@ def sig_int_handler(_: int, __: Any) -> None:  # pragma: no cover
     default=None,
 )
 
-arg_parser.add_argument(
+typing_options.add_argument(
     '--use-one-literal-as-default',
     help='Use one literal as default value for one literal field',
     action='store_true',
     default=None,
 )
 
-arg_parser.add_argument(
+typing_options.add_argument(
     '--set-default-enum-member',
     help='Set enum members as default values for enum field',
     action='store_true',
     default=None,
 )
 
-arg_parser.add_argument(
+field_options.add_argument(
     '--empty-enum-field-name',
     help='Set field name when enum value is empty (default:  `_`)',
     default=None,
 )
 
 
-arg_parser.add_argument(
+field_options.add_argument(
     '--capitalise-enum-members',
+    '--capitalize-enum-members',
     help='Capitalize field names on enum',
```

Comment:
I agree with the spelling change, but how about also temporarily leaving `--capitalise-enum-member` as a fallback and issuing a warning statement, considering the impact on existing users?

Reply:
I'm not entirely familiar with `argparse`, I was under the impression that this would just add an additional way to specify the same flag, similar to `('-h', '--help', ...)`? Let me know if I'm mistaken!

I just tested with both spellings and was able to specify either one to achieve the same result

Reply:
I'm sorry, I misunderstood your change :sweat_smile: 
Looks good.